File: 0783/783.py
class Solution:
    def minimum_distance_between_bst_nodes(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.minimum_distance_between_bst_nodes(root.left), self.minimum_distance_between_bst_nodes(root.right))

File: 0783/783.go
func minimumDistanceBetweenBstNodes(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumDistanceBetweenBstNodes(root.Left)
	r := minimumDistanceBetweenBstNodes(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}